portfolio/src/main/webapp/style.css

```diff
@@ -5,7 +5,7 @@
   width: 650px;
 }
 
-#greeting-container {
+#funFact-container {
```

Comment:
super nit: fun-fact-container 

Super nit on style, but good to remain consistent and have these guidelines become second nature when coding :) Keeps things readable once you start working on larger codebases.

All code should be in lowercase.
(https://google.github.io/styleguide/htmlcssguide.html#Capitalization)

Separate words in ID and class names by a hyphen.
(https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters)

Reply:
Interesting! I've always used the camel case naming convention so that is good to know